metadata-ingestion/src/datahub/ingestion/source/snowflake/snowflake_utils.py

```diff
@@ -37,6 +37,9 @@ def get_dataset_identifier(
     ) -> str:
         ...
 
+    def get_dataset_identifier_from_qualified_name(self, qualified_name: str) -> str:
```

Comment:
Does this have any implementation or why it is just ...?

Reply:
Hey, this Protocol class is only for mypy. This is required if using mixin classes, without actually doing inheritance. Reference  - https://mypy.readthedocs.io/en/latest/more_types.html#mixin-classes

Reply:
Can you add this comment in the code as well?

Reply:
yes

Reply:
done